captain/utils/test_sequencer/run_test_sequence.py

```diff
@@ -240,18 +245,30 @@ def reverse_id(test_name) -> str:
             )
             node.is_saved_to_cloud = True
             logger.info(f"{test_name}: Uploaded to cloud")
-        except KeyError as err:
-            # TODO: Return error to user
+        except KeyError:
```

Comment:
Just to verify, does `client.upload` throw `KeyError` if a test was not found in Cloud under given project?

Reply:
Yes!